src/udev/udevd.c

```diff
@@ -992,7 +992,7 @@ static int on_uevent(sd_device_monitor *monitor, sd_device *dev, void *userdata)
         }
 
         /* we have fresh events, try to schedule them */
-        event_queue_start(manager);
+        (void) event_queue_start(manager);
```

Comment:
Please drop the (void) casts, as discussed in https://github.com/systemd/systemd/pull/20138#discussion_r669340803.

Reply:
OK, at least tentatively.